Stellar-contract.x

```diff
@@ -273,7 +288,9 @@ case SCO_BYTES:
     opaque bin<SCVAL_LIMIT>;
 case SCO_CONTRACT_CODE:
     SCContractCode contractCode;
-case SCO_ACCOUNT_ID:
-    AccountID accountID;
+case SCO_ADDRESS:
+    SCAddress address;
+case SCO_NONCE_KEY:
+    SCAddress nonceAddress;
```

Comment:
Where can I read more about what a nonce key is, and why it contains an address? I've missed something on how this works and don't understand what it is or why it contains an address.

Reply:
I think for the first iteration we'd go with the nonce that works the same way as in the current auth (i.e. it's keyed by contract id + address). Going forward I think we have an agreement to switch to temporarily stored nonces. But that wouldn't make much sense before we have temp entry support.